source/fundamentals/aggregation.txt

```diff
@@ -179,6 +179,59 @@ The above aggregation should produce the following results:
    {"_id": 4, "count": 2}
    {"_id": 5, "count": 1}
 
+Explain Aggregation Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To explain the execution plan of the operation, append the :java-docs:`explain()
```

Comment:
```suggestion
To explain the execution plan of the operation, call the :java-docs:`explain()
```

Issue: I think this should say "call the explain method". I don't believe you can "append a method" (you can call a method on the item returned from another method in a method chain, but I believe appending as a verb is reserved for adding items to the end of array-like data structures). 

I think this paragraph should be rephrased such that it is focused on the ``FindIterable`` object returned from  the call to the ``MongoCollection.aggregate()`` method rather than the `explain()` method appended to `aggregate()`.